chain/cosmos/chain_node.go

```diff
@@ -225,14 +225,28 @@ func (tn *ChainNode) SetTestConfig(ctx context.Context) error {
 
 	c["rpc"] = rpc
 
-	return testutil.ModifyTomlConfigFile(
+	if err := testutil.ModifyTomlConfigFile(
 		ctx,
 		tn.logger(),
 		tn.DockerClient,
 		tn.TestName,
 		tn.VolumeName,
 		"config/config.toml",
 		c,
+	); err != nil {
+		return err
+	}
+
+	a := make(testutil.Toml)
+	a["minimum-gas-prices"] = tn.Chain.Config().GasPrices
+	return testutil.ModifyTomlConfigFile(
+		ctx,
+		tn.logger(),
+		tn.DockerClient,
+		tn.TestName,
+		tn.VolumeName,
+		"config/app.toml",
+		a,
```

Comment:
This isn't related to the forward middleware, but is needed to spin up a Gaia v8 chain due to changes in the SDK?